### PR TITLE
pw-cli: add page

### PR DESCRIPTION
**Pull request by mmatous**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
0.3.50

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/pw-cli.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- List all nodes (sinks and sources) along with their IDs:
+
+`pw-cli list-objects Node`
+
+- Print information about an object with the specified ID:
+
+`pw-cli info {{4}}`
+
+- Print all objects' information:
```
Comment (CleanMachine1) [inline pages/linux/pw-cli.md:14]:
```suggestion
- Print all object information:
```

File: pages/linux/pw-cli.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# pw-cli
+
+> The PipeWire Command Line Interface.
+> More information: <https://docs.pipewire.org/page_man_pw_cli_1.html>.
+
+- List all nodes (sinks and sources) along with their IDs:
```
Comment (EmilySeville7cfg) [inline pages/linux/pw-cli.md:6]:
```suggestion
- Print all nodes (sinks and sources) along with their IDs:
```

File: pages/linux/pw-cli.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`pw-cli list-objects Node`
+
+- Print information about an object with the specified ID:
```
Comment (EmilySeville7cfg) [inline pages/linux/pw-cli.md:10]:
```suggestion
- Print information about an object with a specific ID:
```

File: pages/linux/pw-cli.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# pw-cli
+
+> The PipeWire Command Line Interface.
+> More information: <https://docs.pipewire.org/page_man_pw_cli_1.html>.
+
+- List all nodes (sinks and sources) along with their IDs:
+
+`pw-cli list-objects Node`
```
Comment (EmilySeville7cfg) [inline pages/linux/pw-cli.md:8]: What is `Node` here? Is it a placeholder or not? I don't see any info about `Node` [here][manned]. I expected smth like `Node` or `list-objects [arguments...]`.

[manned]: https://manned.org/pw-cli.1#head6

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Just some suggestions and one question. 😄

File: pages/linux/pw-cli.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# pw-cli
+
+> The PipeWire Command Line Interface.
+> More information: <https://docs.pipewire.org/page_man_pw_cli_1.html>.
```
Comment (EmilySeville7cfg) [inline pages/linux/pw-cli.md:4]: @navarroaxel, @CleanMachine1, what about using [manned.org][doc]? The provided link referrers to a page with not user-friendly formatted page

[doc]: https://manned.org/pw-cli.1#head6

File: pages/linux/pw-cli.md (hunk 1 above)
Comment (mmatous) [reply pages/linux/pw-cli.md:14]: It doesn't print all information about single object but information about every object. All nodes and devices.

File: pages/linux/pw-cli.md (hunk 4 above)
Comment (mmatous) [reply pages/linux/pw-cli.md:8]: It's a keyword. A type. Possible values are `pw-cli list-objects Node` and `pw-cli list-objects Device`.

Comment (EmilySeville7cfg) [reply pages/linux/pw-cli.md:8]: What about this change:

```suggestion
`pw-cli list-objects {{Node|Device}}`
```

? You can also modify your description as: `List all nodes/devices (sinks and sources) along with their IDs:`.

Comment (mmatous) [reply pages/linux/pw-cli.md:8]: It'd be `nodes (sinks and sources)/devices`, sources and sinks are types of node. 

After taking a better look at it, possible values are anything from https://docs.pipewire.org/page_objects_design.html. Considering it's too much to list, I'd stick with the original text. Users most likely want to operate on nodes.

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM

**Pull request merged**